### DISCUSSION TITLE
Add an option to shortcut slider shapes in lazer (osu!catch)

### DISCUSSION THREAD

**Discussion by Du5tOsu** [ppy/osu#24787] [Ideas]
Hi im a mapper and bn for osu catch and recently played around with the editor a bit. Its amazing but noticed one thing that I think would be awesome to have.

Currently the small droplets in osu!catch are randomly generated. That can lead to some weird droplet placement which sometimes makes certain patterns or long sliders harder to properly acc than intended. For example sliders can have the first small droplet appear almost above the first fruit when the mapper intended to do a curved slider. The only way to solve this currently is adding anchor points in osu! and in osu!lazer. 

I saw https://github.com/ppy/osu/discussions/13534#discussioncomment-985454 and the problem with backwards compatibility with the legacy editor. Im by no means a programmer but maybe the workaround ekr mentioned can be added.

What I mean by that: How about adding a shortcut to save certain shapes? The same curves and straight sliders are used a lot throughout a map so having them appear with a short keypress helps a lot. Would be a big qol improvement.

**Comment by stardustprism** [comment]
If possible, this should be stay as an option more rather than forcing every sliders form in shape, I'm not a big fan of derandomizer

**Comment by Du5tOsu** [comment]
Saw a lot of editor changes recently, so im giving it a smol bump

**Comment by osuSpectator** [comment]
agree with stardust prism that it should be optional but would be a good addition regardless

**Comment by snalgae** [comment]
I think the best option is being able to create any curves in the editor instead, but someone needs to figure out math for compatibility
